models/pagination.go

```diff
@@ -9,11 +9,17 @@ type Filter struct {
 type Pagination struct {
 	limit   int
 	offset  int
+	sort    string
 	filters []Filter
 }
 
-func NewPagination(filters ...Filter) *Pagination {
+func NewPagination(sort string, filters ...Filter) *Pagination {
+	if sort == "default" {
```

Comment:
I would move this here: https://github.com/ugent-library/deliver/pull/121/files#diff-a0db2ebda5006de2536a674cc233a72545f1439677ffafe4207eafede50f840cR48

Reply:
See https://github.com/ugent-library/deliver/pull/121#discussion_r1494509845